modules/nf-core/modules/ucsc/bedclip/main.nf

```diff
@@ -1,9 +1,8 @@
-def VERSION = '377' // Version information not provided by tool on CLI
-
 process UCSC_BEDCLIP {
```

Comment:
same issue here: https://github.com/nf-core/modules/blob/master/modules/ucsc/bedclip/meta.yml
and I think that's actually an issue I noticed in cutandrun cf https://github.com/nf-core/cutandrun/pull/159
I'll make an issue on tools